wiki/release/release-process.md

```diff
@@ -303,17 +287,41 @@ https://github.com/apache/pulsar/releases/tag/v2.X.0-candidate-1
 Pulsar's KEYS file containing PGP keys we use to sign the release:
 https://dist.apache.org/repos/dist/dev/pulsar/KEYS
 
+Docker images:
```

Comment:
If we add this part. Let's move the first part of 9.A forward, before this Vote. And remove 9.B.

Reply:
I have linked `#9-publish-docker-images` in the NOTE below. There is no need to move 9.A here, because we only need to build the images and push them to the private account here. But 9.A include one more step:

> Ask a person with access to apachepulsar org in [hub.docker.com](http://hub.docker.com/) to copy the docker images from your (release manager) personal acccount to apachepulsar org.

Reply:
So I refers to "the first part of 9.A". We can split step 9 to "Publish release candidate docker image" and "Publish docker image".
Referring to a step forward make the process kind of out-of-order.